### ISSUE TITLE
Adding R stats package functions

### ISSUE THREAD

**Issue by gasparl** [WillemSleegers/tidystats#18]
I made a list of all the NHST ("`htest`") functions from the [R stats package](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/00Index.html) that are not yet implemented for tidystats:

- [x] ansari.test
- [x] bartlett.test
- [x] binom.test
- [x] box.test
- [x] fligner.test
- [x] friedman.test
- [x] kruskal.test
- [x] ks.test
- [x] mantelhaen.test
- [x] mauchly.test
- [x] mcnemar.test
- [x] mood.test
- [x] poisson.test
- [x] pp.test
- [x] prop.test
- [x] prop.trend.test
- [x] quade.test
- [x] shapiro.test
- [x] var.test

I'd start gradually implementing these.

**Comment by WillemSleegers** [comment]
Thanks @gasparl.

Some of these I have implemented, such as kruskal.test, ks.test, var.test (see the prep_htest.r file). That file also contains a list of other analyses that I don't see in this list but might be possible to add support for.

**Comment by gasparl** [comment]
Ah right, sorry, regarding `htest` tidystats implementations I was looking at the `master` branch.

As for the three pairwise comparisons missing from my list, (`pairwise.t.test`, `pairwise.prop.test`, `pairwise.wilcox.test`), these have only p values returned (in matrixes), so I didn't think they are very applicable for tidystats reporting. But maybe this can be discussed later, or you can let me know if you have an idea for how this could/should be reported.

**Comment by WillemSleegers** [comment]
They could be handled like we handle the results of `confint()`. We add a class to the matrix in `add_stats()` and then parse it in `tidy_stats()`.

**Comment by gasparl** [comment]
This is now all done; if it looks okay to you @WillemSleegers, you can close it.

For each test, I tried to find the best way of reporting, but there are quite a few where I had to make rather arbitrary choices (e.g., what symbols to use for the estimates of rates and ratios). If there was no clear best practice, I generally stuck to R stats' original notation (e.g., `K²` instead of `χ²` for Bartlett test). I guess future `tidystats` users with more familiarity/expertise with any given test can also just let us know if there are better choices.

Also, for now I used my `models_equal` function only for the tests for which I added support. If you want, I can add it to all `htest` function testings. (I could also add it to all other classes, but in those cases there are less tests for each, so maybe it's more readable as it is.)

(The related minor changes in the Google Docs add-in can be checked [here](https://github.com/gasparl/tidystats-Google-Docs-add-in/compare/f748031f3c38c7e1f04c36e661c55a001c270063..5e669ab1c12659cacf94045cd3b9e84ebb1fa1b0) – I guess the MS Word add-in would require the same changes.)

**Comment by WillemSleegers** [comment]
I copied the code to my fork and adapted the code a bit. I still need to adapt the add-in code and manually check whether everything works.

**Comment by WillemSleegers** [comment]
I now manually added the pairwise tests as well. I re-wrote the `tidy_stats()` code for that to improve the grouping structure. I created a group called "Pairs" and in that I store each pair as their own group, with the names combined and separated with a dash. 

(Actually not sure about combining the names, in a previous version I stored the names of pairs as a group with name1 and name2 to store the variable names in).
